resources/js/components/IndexField.vue

```diff
@@ -1,14 +1,45 @@
 <template>
-    <a :href="`tel:+${unmaskedValue}`" class="no-underline text-primary"> {{ field.value }} </a>
-</template>
-
-<script>
-export default {
+    <div :class="`text-${field.textAlign}`">
```

Comment:
I think we need to copy the unmasked value (without spaces), so you can copy the nova components `js/fields/Index/EmailField.vue`